central/role/datastore/singleton.go

```diff
@@ -155,12 +155,13 @@ var vulnReportingDefaultRoles = map[string]roleAttributes{
 		postgresID:  vulnReporterPermissionSetID,
 		description: "For users: use it to create and manage vulnerability reporting configurations for scheduled vulnerability reports",
 		resourceWithAccess: []permissions.ResourceWithAccess{
-			// TODO: ROX-13888 Replace VulnerabilityReports with WorkflowAdministration.
-			permissions.View(resources.VulnerabilityReports),   // required for vuln report configurations
-			permissions.Modify(resources.VulnerabilityReports), // required for vuln report configurations
-			permissions.View(resources.Role),                   // required for scopes
-			permissions.View(resources.Image),                  // required to gather CVE data for the report
-			permissions.View(resources.Integration),            // required for vuln report configurations
+			permissions.View(resources.WorkflowAdministration),   // required for vuln report configurations
+			permissions.Modify(resources.WorkflowAdministration), // required for vuln report configurations
+			permissions.View(resources.Role),                     // required for scopes
```

Comment:
We do not need `Role` permissions are required anymore.

Reply:
As of right now, `Role` is still required in the service layer implementation for report configurations. Is the migration from `ReportConfiguration` to `WorkflowAdministration` something we intend to do with MVP or is that a post-MVP task?

Reply:
With collections we do not require `Role` because `Role` was required by `AccessScopes` which is being replaced.

Reply:
If we cut a release without collections being enabled `Role` will still be required for the flow to work. It seems to me like it would be a good idea to leave it in there until it's fully deprecated, unless there's some significant reason why it should be removed

Reply:
Those perms need to be added conditionally based on the feature flag.

Reply:
Couldn't accomplish this in place because go doesn't like ternary. I added some filtering code before the object is used and will verify with the image that the values are excluded before merging